lib/sparse/src/index/inverted_index/inverted_index_mmap.rs

```diff
@@ -113,6 +121,9 @@ impl InvertedIndexMmap {
         // file index data
         Self::save_posting_headers(&mut mmap, inverted_index_ram, total_posting_headers_size);
         Self::save_posting_elements(&mut mmap, inverted_index_ram, total_posting_headers_size);
+        if file_length > 0 {
```

Comment:
why is this necessary now?

Reply:
Because on macos and windows `flush` panics when file is empty =(
It can be empty when we create default index